configure.ac

```diff
@@ -178,6 +178,42 @@ if test x"$try_xmlrpc_fallback" = xtrue; then
     AC_SUBST(XMLRPC_LIBS)
 fi
 
+dnl ---------------------------------------------------------------------------
+dnl - Check for LIBCURL
+dnl ---------------------------------------------------------------------------
+PKG_CHECK_MODULES([LIBCURL], [libcurl], [],
```

Comment:
You can keep it simple and assume that platforms have proper pkg-config support for necessary libraries:

```
dnl ---------------------------------------------------------------------------
dnl - Check for jansson and libcurl for ipa-join
dnl ---------------------------------------------------------------------------
PKG_CHECK_MODULES([JANSSON], [jansson])
PKG_CHECK_MODULES([LIBCURL], [libcurl])
```